packages/binutils/binutils.spec

```
@@ -0,0 +1,70 @@
+Name: %{_cross_os}binutils
+Version: 2.35.2
+Release: 1%{?dist}
+Summary: A set of GNU tools for working with binaries
+URL: https://sourceware.org/binutils
+License: GPL-3.0-or-later
+Source0: https://ftp.gnu.org/gnu/binutils/binutils-%{version}.tar.xz
+Requires: %{_cross_os}libz
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libz-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using binutils
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n binutils-%{version} -p1
+
+%build
+# Fail if the SDK version is different than the one provided in the image
+test %{version} = $(%{__ld} -v | cut -d ' ' -f 5) || exit 1
```

nit: prefer `[` to `test`.

I'd also recommend using `awk` to get the last field rather than using the field count - this one gets the version field for our build and the default `/usr/bin/ld`:

```suggestion
[ %{version} = $(%{_cross_target}-ld -v | | awk '{print $NF}') ] || exit 1
```